dashboard/config.yml

```diff
@@ -4,4 +4,15 @@ server:
     port: 8000
   adminConnectors:
   - type: http
-    port: 8001
\ No newline at end of file
+    port: 8001
+deployer:
+  host: ${DEPLOYER_HOST}
```

Comment:
In this way we are still substituting those values using ENV VAR, which is fine but not ideal.
When I suggested to use Dropwizard configuration I was expecting to have a `config.yml.template` where values were replaced by Brooklyn when known, but we'd need a dashboard entity for that, so let's keep it as is.

Probably the idiomatic way in Dropwizard is to define your own `ConfigurationSourceProvider` but not sure.